src/org/folio/Constants.groovy

```diff
@@ -8,7 +8,7 @@ class Constants {
     static String AWS_S3_DATA_EXPORT_ID = 'ci-data-export-s3'
     static String AWS_S3_POSTGRES_BACKUPS = 'manage-postgres-db-backups-s3'
     static String AWS_EKS_VPC_NAME = 'folio-rancher-vpc'
-    static String AWS_EKS_ADMIN_USERS = 'oleksandrhaimanov,kseniiadubniak,vitaliiustalu,volodymyrkartsev,vasilikapylou,renatsafiulin,tarasspashchenko,stanislav'
+    static String AWS_EKS_ADMIN_USERS = 'oleksandrhaimanov,kseniiadubniak,volodymyrkartsev,vasilikapylou,renatsafiulin,tarasspashchenko,stanislav'
```

Comment:
If you edit this var, let add Eldiar and Guram here)

Reply:
Sure)